jest.strict.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  displayName: 'strict',
+  collectCoverageFrom: ['<rootDir>/app/scripts/controllers/permissions/*.js'],
+  coverageDirectory: './jest-coverage/strict/',
+  coverageReporters: ['json', 'lcov', 'text', 'clover'],
+  coverageThreshold: {
+    './app/scripts/controllers/permissions/*.js': {
```

Comment:
Does this have to be a separate config? We could merge it with the base config, since the only difference is the coverage threshold.

Reply:
No, they don't have to be separate: 5bfbe8e15